packages/indy-vdr/tests/helpers.ts

```diff
@@ -2,13 +2,14 @@ import type { IndyVdrPoolService } from '../src/pool/IndyVdrPoolService'
 import type { AgentContext, Key } from '@aries-framework/core'
 
 import { KeyType } from '@aries-framework/core'
-import { AttribRequest, NymRequest } from '@hyperledger/indy-vdr-shared'
+import { AttribRequest, NymRequest, indyVdr } from '@hyperledger/indy-vdr-shared'
```

Comment:
This should also import `indyVdr` from `indy-vdr-nodejs` instead of `shared`

Reply:
General strategy on where to import from:
- non-test files should always be platform agnostic (so `-shared`).
- test files need a specific implementation, and we're running in Node.JS (so `-nodejs`).
- We only need to have documentation for the `-react-native` dependency in the module config